delphi-frontend/src/main/antlr3/au/com/integradev/delphi/antlr/Delphi.g

```diff
@@ -720,10 +759,13 @@ expressionOrRangeList        : (expressionOrRange (','!)?)+
 exprOrRangeOrAnonMethodList  : (exprOrRangeOrAnonMethod (','!)?)+
                              ;
 textLiteral                  : textLiteral_ -> ^(TkTextLiteral<TextLiteralNodeImpl> textLiteral_)
+                             | multilineTextLiteral -> ^(TkTextLiteral<TextLiteralNodeImpl> multilineTextLiteral)
                              ;
 textLiteral_                 : TkQuotedString (escapedCharacter+ TkQuotedString)* escapedCharacter*
```

Comment:
Could we use the opportunity of a new text literal rule to rename `textLiteral_` to something less crap - maybe `singleLineTextLiteral`?